frontend/src/router/routes.js

```diff
@@ -116,6 +116,11 @@ export default new Router({
       name: 'Sign Up',
       component: () => import('../views/SignUp')
     },
+    {
+      path: '/verify-email',
```

Comment:
new route eh?

Reply:
Yeah honestly the whole signup flow could probably be in one page tbh. But meh